gov/politeia/piclient/piclient.go

```diff
@@ -15,6 +16,24 @@ import (
 	piapi "github.com/decred/politeia/politeiawww/api/v1"
 )
 
+var (
+	// nullProposalData defines a case scenario where the proposal could have had
+	// a null pointer attached to it.
+	nullProposalData = []byte(`{"proposal": null}`)
```

Comment:
A little fragile if pi starts returning indented or otherwise formatted responses.  I think the safe thing to do is check the pointer on the unmarshalled response.

Reply:
/o\, I totally missed the point earlier. Let me add the fix.